src/app/valid-email-message/valid-email-message.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-valid-email-message',
+  templateUrl: './valid-email-message.component.html',
+  styleUrls: ['./valid-email-message.component.scss']
+})
+export class ValidEmailMessageComponent implements OnInit {
+
+  constructor() { }
```

Comment:
if it's not doing anything, why we need it?

Reply:
This is so we can have the route path /success after user successfully submits their email and it breaks down the huge template on the landing-page component.